ansibullbot/utils/component_tools.py

```diff
@@ -179,32 +178,23 @@ def index_files(self):
         # make a list of names by calling ansible-doc
         checkoutdir = self.gitrepo.checkoutdir
         checkoutdir = os.path.abspath(checkoutdir)
-        cmd = u'. {}/hacking/env-setup; ansible-doc -t module -F'.format(checkoutdir)
-        logging.debug(cmd)
-        (rc, so, se) = run_command(cmd, cwd=checkoutdir)
-        if rc:
-            raise Exception("'ansible-doc' command failed (%s, %s %s)" % (rc, so, se))
-        lines = to_text(so).split(u'\n')
-        for line in lines:
-
-            # compat for macos tmpdirs
-            if u' /private' in line:
-                line = line.replace(u' /private', u'', 1)
-
-            parts = line.split()
-            parts = [x.strip() for x in parts]
-
-            if len(parts) != 2 or checkoutdir not in line:
-                continue
-
-            mname = parts[0]
-            if mname not in self.MODULE_NAMES:
-                self.MODULE_NAMES.append(mname)
-
-            fpath = parts[1]
-            fpath = fpath.replace(checkoutdir + u'/', u'')
+        for root, directories, filenames in os.walk(os.path.join(checkoutdir, 'plugins', 'modules')):
```

Comment:
Not sure if we need this change still.

Reply:
I believe this has been moved out to https://github.com/ansible/ansibullbot/blob/d3274fb73488cfc3a7b1d9321360ea9153fafb96/ansibullbot/utils/git_tools.py#L64-L66

It's no longer here: https://github.com/ansible/ansibullbot/blob/master/ansibullbot/utils/component_tools.py#L293-L302